src/structures/ApplicationCommand.js

```diff
@@ -239,7 +245,8 @@ class ApplicationCommand extends Base {
       (option.required ?? (['SUB_COMMAND', 'SUB_COMMAND_GROUP'].includes(optionType) ? undefined : false)) !==
         existing.required ||
       option.choices?.length !== existing.choices?.length ||
-      option.options?.length !== existing.options?.length
+      option.options?.length !== existing.options?.length ||
+      (option.channelTypes?.length ?? option.channel_types.length) !== existing.channelTypes?.length
```

Comment:
```suggestion
      ((option.channelTypes ?? option.channel_types).length) !== existing.channelTypes?.length
```

Reply:
I was never really a fan of this pattern, but in this case it works.